data/endpoint_data.json

```diff
@@ -0,0 +1,158 @@
+{
+    "comparison": "USA",
+    "metrics": [
+        {
+            "indicator": "GINI INDEX",
+            "ranking": 72,
+            "sortedCountries": ["Armenia", "Azerbaijan", "USA", "Bulgaria", "Georgia", "Greece", "Iran", "Iraq", "Syria"],
```

Comment:
@juanMarceloMartin can we change this to ISO Alpha-3?
Gustavo has already changed the dataset

Reply:
done